packages/loot-core/src/server/main.ts

```diff
@@ -1099,17 +1099,25 @@ function handleSyncError(err, acct) {
   };
 }
 
-handlers['accounts-bank-sync'] = async function ({ id }) {
+handlers['accounts-bank-sync'] = async function ({ id, ids }) {
```

Comment:
Would it make sense to simplify this, so that it only takes an optional list of ids?  If someone wants to sync one, they can pass in a list of 1 account.